crates/modelardb_server/src/optimizer/model_simple_aggregates.rs

```diff
@@ -159,7 +160,7 @@ fn try_new_aggregate_exec(
         ));
     }
 
-    let model_based_aggregate_exprs = try_rewrite_aggregate_exprs(aggregate_exec)?;
+    let model_based_aggregate_exprs = try_rewrite_aggregate_exprs(aggregate_exec).unwrap();
```

Comment:
Maybe add a comment explaining why unwrap is safe.

Reply:
Good catch, I added `unwrap()` to see what the error message was but any errors that occur must be returned for the optimizer rule to work.